src/App.tsx

```diff
@@ -1,173 +1,120 @@
-import { Loader } from './components/Loader';
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
 
 import './App.scss';
 
-export const App = () => (
-  <div data-cy="app">
-    <nav
-      data-cy="nav"
-      className="navbar is-fixed-top has-shadow"
-      role="navigation"
-      aria-label="main navigation"
-    >
-      <div className="container">
-        <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
-            Home
-          </a>
-
-          <a
-            className="navbar-item has-background-grey-lighter"
-            href="#/people"
-          >
-            People
-          </a>
+import {
+  NavLink, Routes, Route, Navigate,
+} from 'react-router-dom';
+import { Loader } from './components/Loader';
+import { Person } from './types/Person';
+import { getPeople } from './api';
+import { PeoplePage } from './components/PeoplePage/PeoplePage';
+
+type Props = {
+  to: string;
+  text: string;
+};
+
+export const PageNavLink: React.FC<Props> = ({ to, text }) => {
+  return (
+    <>
+      <NavLink
+        className={({ isActive }) => classNames(
+          'navbar-item',
+          { 'has-background-grey-lighter': isActive },
+        )}
+        to={to}
+      >
+        {text}
+      </NavLink>
+    </>
+  );
+};
+
+export const App: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [error, setError] = useState<boolean>(false);
+
+  const getpeople = async () => {
+    const fetchPeople = await getPeople();
+
+    setIsLoading(true);
```

Comment:
```suggestion
    setIsLoading(true);
    
    const fetchPeople = await getPeople();
```
it should fix this test
![Screenshot from 2023-05-13 16-30-49](https://github.com/mate-academy/react_people-table-basics/assets/33029864/180011ba-647f-4f36-9d03-6eded300a9ed)